docs/tutorial/part-two/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Introduction to Styling in Gatsby
+title: Pengenalan Styling di Gatsby
```

Comment:
Title change with “judul”

Reply:
Also, please add italic format in styling

Reply:
got it

Reply:
should we really translate title into "judul"? After i checked other translations, they don't translate it.

Reply:
For now, i think we can leave it.